### PR TITLE
can specify port even in native mode

### PR DESCRIPTION
**Pull request by retsyo**
orennir points out that it is useful to specify port even in navtie mode. https://github.com/zauberzeug/nicegui/discussions/1457

### CODE REVIEW

File: nicegui/ui_run.py [hunk 1]
```diff
@@ -121,10 +121,11 @@ def run(*,
     if native:
         show = False
         host = host or '127.0.0.1'
-        port = native_module.find_open_port()
+        port = port or native_module.find_open_port()
         width, height = window_size or (800, 600)
         native_module.activate(host, port, title, width, height, fullscreen, frameless)
     else:
+        port = 8080
```
Comment (rodja) [inline nicegui/ui_run.py:128]: How can a user define the port when not in native mode? It seems to always be set to 8080.

File: nicegui/ui_run.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def run(*,
     Most of them only apply after stopping and fully restarting the app and do not apply with auto-reloading.
 
     :param host: start server with this host (defaults to `'127.0.0.1` in native mode, otherwise `'0.0.0.0'`)
-    :param port: use this port (default: `8080`)
+    :param port: use this port (default to `8080` not in native mode. If not specify, NiceGUI searches for an open port in native mode)
```
Comment (rodja) [inline nicegui/ui_run.py:56]: The formulation "default to `8080` not in native mode" is not descriptive enough.

**Review by rodja — CHANGES_REQUESTED** [review]

File: nicegui/ui_run.py (hunk 2 above)
Comment (retsyo) [reply nicegui/ui_run.py]: I changed it to 
```
:param port: use this port (If not specify, default to `8080` not in native mode. If not specify, NiceGUI searches for an open port in native mode)
```

File: nicegui/ui_run.py (hunk 1 above)
Comment (retsyo) [reply nicegui/ui_run.py]: I changed it to
```python
port = port or 8080
```

**Comment by retsyo** [comment]
I am not familiar with the operations on www.github.com. Since my previous modification is not good enought, I edit code in my own fork again. Is it a right way?

File: nicegui/ui_run.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ def run(*,
     Most of them only apply after stopping and fully restarting the app and do not apply with auto-reloading.
 
     :param host: start server with this host (defaults to `'127.0.0.1` in native mode, otherwise `'0.0.0.0'`)
-    :param port: use this port (default: `8080`)
+    :param port: use this port (If not specify, default to `8080` not in native mode. If not specify, NiceGUI searches for an open port in native mode)
```
[the remark this answers is not in the document]
Comment (rodja) [reply nicegui/ui_run.py:56]: Yes that sounds good!

File: nicegui/ui_run.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
 
 def run(*,
         host: Optional[str] = None,
-        port: int = 8080,
+        port: int = 0,
```
[the remark this answers is not in the document]
Comment (retsyo) [reply nicegui/ui_run.py:25]: yes, it is better

**Review by falkoschindler — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**